### PR TITLE
Fix: moving function attributes inside samples, to make them self-descriptive

### PR DESCRIPTION
**Pull request by BigBlackWolf**
## Description

- moving function attributes inside samples, to make them self-descriptive

## Checklist
- [ ] I have followed [Contributing Guidelines from CONTRIBUTING.MD](https://github.com/GoogleCloudPlatform/golang-samples/blob/main/CONTRIBUTING.md)
- [ ] **Tests** pass:   `go test -v ./..` (see [Testing](https://github.com/GoogleCloudPlatform/golang-samples/blob/main/CONTRIBUTING.md#testing))
- [ ] **Code formatted**:   `gofmt` (see [Formatting](https://github.com/GoogleCloudPlatform/golang-samples/blob/main/CONTRIBUTING.md#formatting))
- [ ] **Vetting** pass:   `go vet` (see [Formatting](https://github.com/GoogleCloudPlatform/golang-samples/blob/main/CONTRIBUTING.md#formatting))
- [ ] These samples need a new **API enabled** in testing projects to pass (let us know which ones)
- [ ] These samples need a new/updated **env vars** in testing projects set to pass (let us know which ones)
- [ ] This sample adds a new sample directory, and I updated the [CODEOWNERS file](https://github.com/GoogleCloudPlatform/golang-samples/blob/main/.github/CODEOWNERS) with the codeowners for this sample
- [ ] This sample adds a new **Product API**, and I updated the [Blunderbuss issue/PR auto-assigner](https://github.com/GoogleCloudPlatform/golang-samples/blob/main/.github/blunderbuss.yml) with the codeowners for this sample
- [ ] Please **merge** this PR for me once it is approved

### CODE REVIEW

File: vertexai/multimodal-video/multimodalvideo_test.go [hunk 1]
```diff
@@ -25,12 +25,10 @@ func Test_generateMultimodalContent(t *testing.T) {
 	tc := testutil.SystemTest(t)
 
 	buf := new(bytes.Buffer)
-	prompt := "What is in this video?"
-	video := "gs://cloud-samples-data/video/animals.mp4"
 	location := "us-central1"
 	modelName := "gemini-1.5-flash-001"
 
-	err := generateMultimodalContent(buf, prompt, video, tc.ProjectID, location, modelName)
+	err := generateMultimodalContent(buf, tc.ProjectID, location, modelName)
```
Comment (rsamborski) [inline vertexai/multimodal-video/multimodalvideo_test.go:31]: fix: please update the test to check the generation response ([example](https://github.com/GoogleCloudPlatform/golang-samples/blob/main/vertexai/multimodal-video-with-audio/multimodalvideoaudio_test.go#L42-L52))

File: vertexai/multimodal/multimodal_test.go [hunk 2]
```diff
@@ -25,12 +25,10 @@ func Test_generateMultimodalContent(t *testing.T) {
 	tc := testutil.SystemTest(t)
 
 	buf := new(bytes.Buffer)
-	prompt := "describe what is in this picture"
-	image := "gs://generativeai-downloads/images/scones.jpg"
 	location := "us-central1"
 	modelName := "gemini-1.5-flash-001"
 
-	err := generateMultimodalContent(buf, prompt, image, tc.ProjectID, location, modelName)
+	err := generateMultimodalContent(buf, tc.ProjectID, location, modelName)
```
Comment (rsamborski) [inline vertexai/multimodal/multimodal_test.go:31]: fix: ditto, please test the response as well

File: vertexai/multimodal/multimodal.go [hunk 3]
```diff
@@ -48,11 +44,11 @@ func generateMultimodalContent(w io.Writer, prompt, image, projectID, location,
...
 	// Given an image file URL, prepare image file as genai.Part
 	img := genai.FileData{
-		MIMEType: mime.TypeByExtension(filepath.Ext(image)),
-		FileURI:  image,
+		MIMEType: mime.TypeByExtension(filepath.Ext("scones.jpg")),
+		FileURI:  "gs://generativeai-downloads/images/scones.jpg",
 	}
 
-	res, err := model.GenerateContent(ctx, img, genai.Text(prompt))
+	res, err := model.GenerateContent(ctx, img, genai.Text("describe what is in this picture"))
```
Comment (rsamborski) [inline vertexai/multimodal/multimodal.go:51]: nit: can we start the prompt with capital D?

**Review by msampathkumar — COMMENTED** [review]
LGTM

**Review by subfuzion — APPROVED** [review]

**Pull request merged**